scrapy/utils/reactor.py

```diff
@@ -49,13 +49,21 @@ def __call__(self):
         return self._func(*self._a, **self._kw)
 
 
-def install_reactor(reactor_path):
+def install_reactor(reactor_path, event_loop_path):
```

Comment:
```suggestion
def install_reactor(reactor_path, event_loop_path=None):
```

So that function calls can be cleaner for non-asyncio reactors.